clang/lib/CodeGen/ItaniumCXXABI.cpp

```diff
@@ -1350,7 +1350,7 @@ bool ItaniumCXXABI::classifyReturnType(CGFunctionInfo &FI) const {
   // If C++ prohibits us from making a copy, return by address.
   if (!RD->canPassInRegisters()) {
     auto Align = CGM.getContext().getTypeAlignInChars(FI.getReturnType());
-    FI.getReturnInfo() = ABIArgInfo::getIndirect(Align, /*ByVal=*/false);
+    FI.getReturnInfo() = ABIArgInfo::getIndirect(Align, 0, /*ByVal=*/false);
```

Comment:
Comment parameter name. Maybe it's possible to do better than just hardcoded 0?

Reply:
We could use the target equivalent of `LangAS::Default`, which would at least partially cover one of @rjmccall's suggestions.

Reply:
I'm assuming these should default to DL.getAllocaAddrSpace but I guess that's a separate change that calls for new test coverage

Reply:
I actually went ahead and bit the bullet, switching indirect returns to the AllocaAS (this is at least initially covered by tests here), and leaving other uses of indirect using LangAS::Default. I'm going to have to revisit this around indirect args, but I wasn't brave enough to go and add even more noise to what is already a chunky patch.